### DISCUSSION TITLE
PCL2 用于提供公告的服务是腾讯云的什么服务？

### DISCUSSION THREAD

**Discussion by Silverteal** [Meloong-Git/PCL#3529] [提问] [answered]
就是`pcl2-server-1253424809.file.myqcloud.com?sign=xxxxx`。
看起来像腾讯云的COS，但是腾讯云COS似乎并不提供这种类型的默认域名（无论是当前版本还是历史版本文档中都没有找到`bucket-name-userid.file.myqcloud.com`后面直接加object的这种格式）。签名（`?sign=`）也不像是COS的方式。

**Comment by wuliaodexiaoluo** [comment]
Object Storage 对象存储（你可以理解为是一个只有你自己用的直链网盘）
sign那段是签名密钥，授权给你允许你访问

**Reply by Silverteal** [reply]
你好，请问您读完问题了吗

**Comment by Silverteal** [comment, marked as answer]
又仔细翻了一遍文档，原来是历史版本的COS默认CDN加速域名，2022年开始就不开放新用户接入了。
